department-of-veterans-affairs/va.gov-cms
# VAGOV-5111: Enable workbench access for office content type.

QA
* Go to admin/config/workflow/workbench_access/access_scheme/section/edit and note that Office (and all other node types) have their "Owner" field enabled as a Workbench Access field.
